roxctl/sensor/generate/generate.go

```diff
@@ -46,11 +54,19 @@ type sensorGenerateCommand struct {
 
 func defaultCluster() *storage.Cluster {
 	return &storage.Cluster{
+		AdmissionControllerEvents:  true,
```

Comment:
@janisz I know we discussed this is not necessary (since the defaults have been changed when the option flag is registered).
However, I added this as a forward looking safety net. When the marked as deprecated options are actually removed, I do not want the developer to forget doing this at that time - it will be needed then, without it, the defaults would still be incorrect.